classes/shaman.lua

```diff
@@ -133,6 +183,58 @@ local function registerClass(self)
         self:RegisterAura("elemental_focus_1", 1, 16246, "echo_of_the_elements", "Left", 1, 255, 255, 255, true);
         self:RegisterAura("elemental_focus_2", 2, 16246, "echo_of_the_elements", "Left + Right (Flipped)", 1, 255, 255, 255, true);
     end
+    
+    if self.IsCata() then
+        self:RegisterAura("elemental_focus_1", 1, 16246, "genericarc_05", "Left", 1.5, 255, 255, 255, true);
+        self:RegisterAura("elemental_focus_2", 2, 16246, "genericarc_05", "Left + Right (Flipped)", 1.5, 255, 255, 255, true);
+        local lavaBurstCata = 51505;
+        self:RegisterAura("lava_surge", 0, lavaBurstCata, "imp_empowerment", "Left + Right (Flipped)", 1, 255, 255, 255, true, {lavaBurstCata}, true);
+        self:RegisterCounter("lava_surge");
+        if (not RollingThunderHandler.initialized) then
+            RollingThunderHandler:init();
+        end
+        for lightningShieldStacks=6,9 do
+            local auraName = "fulmination_"..lightningShieldStacks;
+            local scale = 0.5 + 0.1 * (lightningShieldStacks - 5); -- 60%, 70%, 80%

```

Comment:
The comment indicates 60% to 80%, but it goes up to 90% at 9 stacks:
$`0.5 + 0.1*(stacks - 5) = 0.5 + 0.1*(9-5) = 0.5 + 0.1*4 = 0.5 + 0.4 = 0.9`$

Reply:
same as above, it's 0.4 + 0.1 * (stacks) now